src/main/java/fi/metatavu/metaform/server/rest/RealmsApiImpl.java

```diff
@@ -282,8 +311,43 @@ public Response deleteReply(String realmId, UUID metaformId, UUID replyId) throw
 
   @Override
   public Response replyExport(String realmId, UUID metaformId, UUID replyId, String format) throws Exception {
-    // TODO: Add in issue #46 (https://github.com/Metatavu/metaform-api/issues/46)
-    return null;
+    // TODO: Permission check
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=49&issues=AWWBxMGxXKENQn1S5jLU&open=AWWBxMGxXKENQn1S5jLU)

[ISSUE_KEY]: AWWBxMGxXKENQn1S5jLU